src/smc-webapp/misc_page.coffee

```diff
@@ -60,9 +60,9 @@ exports.is_enter       = (e) -> e.which is 13 and not e.shiftKey
 exports.is_ctrl_enter  = (e) -> e.which is 13 and e.ctrlKey
 exports.is_escape      = (e) -> e.which is 27
 
-{join} = require('path')
-exports.APP_BASE_URL = window?.app_base_url ? ''
-exports.BASE_URL = if window? then "#{window.location.protocol}//#{join(window.location.hostname, window.app_base_url ? '')}" else theme.DOMAIN_NAME
+base_url_lib = require("./misc/base-url")
+exports.APP_BASE_URL = exports.APP_BASE_URL
```

Comment:
Surely this line should be `exports.APP_BASE_URL = base_url_lib.APP_BASE_URL` and similar below?

Reply:
yes, how silly is that? thank you for spotting that.